### ISSUE TITLE
The "X" button for cleaning input content cannot be focused on with tab order

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1942]
Priority:
P2

URL: http://clarity-ks.surge.sh/datagrids

Reproduced Steps:
Go to Datagrids page with IE.
Click the filter button on the page.
Input some text in the filter search button, please refer to screen shot
Use tab order to point to the “X” button which used to clean search.

Actual Results:
The "X" button for cleaning input content cannot be focused on with tab order.

Expected Results:
The "X" button for cleaning input content should can be focused on with tab order.

Violation of Section 508 Standards: 
1194.21(a) When software is designed to run on a system that has a keyboard, product functions shall be executable from a keyboard where the function itself or the result of performing a function can be discerned textually. 

Screen shot:

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/0bb87a6f-e917-40dd-a476-a51876e6b172)

Note:
1. This issue is only reproduced with IE browser.
2. This issue also occurred on Forms, Login, Vertical Nav tab.

**Comment by youdz** [comment]
Wait, this "X" input is an IE thing, we don't have one. This isn't even DOM, it's internal browser implementation, we can't make the "X" focusable.

I'm closing this because it makes no sense as far as I can tell.
